src/components/HearingImageList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import HearingImage from './HearingImage';
+
+class HearingImageList extends React.Component {
+  render() {
+    const {images} = this.props;
+    return (<div>
+      {images.map((image, index) => <HearingImage data={image} key={index}/>)}
```

Comment:
Maybe use the image URL as the key?

I mean, I doubt using an index will do the Right Thing during DOM reconciliation, as if there were, say, images "0", "1" and "2", and you'd delete the one formerly called "1", you'd then get "0" and "1", and React would think "2" has gone away.